products/sle/main.pm

```diff
@@ -769,7 +769,9 @@ sub load_x11tests() {
         loadtest "x11/reboot_gnome";
     }
     loadtest "x11/desktop_mainmenu";
-    loadtest "x11/shutdown";
+    unless (get_var('ROLLBACK_AFTER_MIGRATION')) {
```

Comment:
I guess it's not quite clear why you want to exclude the shutdown here. maybe a short comment?